content/concepts/long-polling.textile

```diff
@@ -0,0 +1,149 @@
+
+As per the readme, php-long-polling is "a very simple demonstration of long-polling with AJAX (jQuery) and PHP ... This is an improved, cleaned and documented fork of (php-ajax-long-polling)[https://github.com/lincolnbrito/php-ajax-long-polling]".
+
+As the code for this one is extremely simple you can take a look at the "implementation":https://github.com/panique/php-long-polling/blob/master/server/server.php directly - it's fairly self-explanatory.
+
+You can find out more about this library from the original "GitHub repo":https://github.com/panique/php-long-polling.
+
+h3(#nodejs-pollymer). Node.js: "Pollymer":https://github.com/fanout/pollymer
+
+From the readme, "Pollymer is a general-purpose AJAX library that provides conveniences for long-polling applications, such as request retries, exponential backoff between requests, randomized request delaying, and workarounds for browser "busy" indications. It also implements multiple transports to ensure cross-domain access works in all major browsers."
+
+Optional extras include support for JSON-P and logging.
+
+```[js]
+var req = new Pollymer.Request();
+req.on('finished', function(code, result, headers) { ... });
+req.on('error', function(reason) { ... });
+var headers = { ... };
+var body = 'some data';
+req.maxTries = 2; // try twice
+req.start('POST', 'http://example.com/path', headers, body);
+```
+
+You can find out more about this library from the original "GitHub repo":https://github.com/fanout/pollymer.
+
+h3(#python-a-simple-comet-server). Python: "A simple COMET server":https://github.com/jedisct1/Simple-Comet-Server
+
+This is a minimalist HTTP long-polling server for the Python language. It supports multiple channels and cross-domain requests. It requires Python &gt;= 2.5 and the "Twisted framework":https://twistedmatrix.com/trac/. Unlike the other libraries mentioned above, this server runs as a kind of broker that sits between the client and the server-side applications. When the server wishes to publish messages to a client, it can make a request to an HTTP(S) endpoint that can be easily hidden behind a firewall, ensuring that is only accessible by server-side processes.
+
+The README for details provides examples for:
+
+* Registering a channel
+* Pushing data to a channel
+* Registering a client
+* Monitoring the server
+* Reading data from channels
+
+You can find out more about this library from the original "GitHub repo":https://github.com/jedisct1/Simple-Comet-Server.
+
+h2(#ably-and-long-polling). Ably and long polling
+
+At Ably we've implemented a robust suite of transports in the client libraries that we make available for communication with our servers, with WebSockets being the primary protocol for realtime communications. However, long polling is fully-supported as an automatic fallback transport when newer standards such as WebSockets are not available on the client-side. Find out more on the "Ably home page":https://ably.io.
```

Comment:
Could users not find out more on a better page such as our platform page which talks about transports etc?